integrations/tensorflow/e2e/keras/BUILD

```diff
@@ -284,8 +285,9 @@ iree_e2e_cartesian_product_test_suite(
     ],
 )
 
+# 'non_hermetic' tests use real model weights to test numerical correctness.
 iree_e2e_cartesian_product_test_suite(
-    name = "imagenet_external_tests",
+    name = "imagenet_non_hermetic_tests",
```

Comment:
This is mentioned in a few scripts. Please update those as well.

Reply:
Thanks!